### PR TITLE
Feature/modal

### PR DESCRIPTION
**Pull request by leticiawanderley**
Default modal
Default

Modal with actions
With actions

### CODE REVIEW

File: src/components/Modal/index.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            )}
+        </div>
+      </ReactModal>
+    )
+  }
+}
+
+Modal.propTypes = {
+  title: PropTypes.string,
+  innerText: PropTypes.string.isRequired,
```
Comment (kevinch) [inline src/components/Modal/index.js]: This should be a `node` i think, like `children` so we can drop in anything.

File: src/components/Modal/index.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  isOpen: PropTypes.bool.isRequired,
```
Comment (kevinch) [inline src/components/Modal/index.js]: This should not be required. The default state for a modal will be _closed_.

File: src/components/Modal/index.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  handleClose: PropTypes.func.isRequired,
+  style: PropTypes.object.isRequired,
```
Comment (kevinch) [inline src/components/Modal/index.js]: Why is _style_ is required?

File: src/components/Modal/index.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  style: PropTypes.object,
+  handlePrimaryAction: PropTypes.func,
+  primaryActionTitle: PropTypes.string,
+  handleSecondaryAction: PropTypes.func,
+  secondaryActionTitle: PropTypes.string,
+}
+
+Modal.defaultProps = {
+  isOpen: false,
+  style: {
```
Comment (klzns) [inline src/components/Modal/index.js]: Se não passar a prop `style` como que fica o layout? Ele não quebra?

File: src/components/Modal/index.js [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+Modal.propTypes = {
+  title: PropTypes.string,
+  innerText: PropTypes.node.isRequired,
+  isOpen: PropTypes.bool,
+  handleClose: PropTypes.func.isRequired,
+  style: PropTypes.object,
+  handlePrimaryAction: PropTypes.func,
+  primaryActionTitle: PropTypes.string,
+  handleSecondaryAction: PropTypes.func,
```
Comment (klzns) [inline src/components/Modal/index.js]: `onClickSecondaryAction`

Comment (klzns) [reply src/components/Modal/index.js]: Ou entao

```js
primaryAction: PropTypes.shape({
  label: PropTypes.string.isRequired,
  onClick: PropTypes.func.isRequired,
})
```

File: src/components/Modal/index.js [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+            )}
+        </div>
+      </ReactModal>
+    )
+  }
+}
+
+Modal.propTypes = {
+  title: PropTypes.string,
+  innerText: PropTypes.node.isRequired,
```
Comment (klzns) [inline src/components/Modal/index.js]: Pode ser `content` ou `description` ao inves de `innerText`?

File: src/components/Modal/index.js [hunk 7]
```diff
@@ -72,22 +67,25 @@ class Modal extends PureComponent {
...
-  primaryActionTitle: PropTypes.string,
-  handleSecondaryAction: PropTypes.func,
-  secondaryActionTitle: PropTypes.string,
+  style: PropTypes.shape({
+    modalheight: PropTypes.string.isRequired,
+    modaloverlay: PropTypes.string.isRequired,
+  }).isRequired,
+  primaryAction: PropTypes.shape({
+    label: PropTypes.string.isRequired,
+    handleClick: PropTypes.func.isRequired,
```
Comment (klzns) [inline src/components/Modal/index.js]: onClick

**Pull request merged**